### ISSUE TITLE
Admin panel functions

### ISSUE THREAD

**Issue by FunctionalMetatable** [FlagClicked/Backend#8]
self explanatory

**Comment by markverb1** [comment]
ban, kick, mute (no comments), block from making tutorials, alerts, delete stuff and block words is most likely what we will need to do first

**Comment by TheColaber** [comment]
Oh right, I need to delete this repo. But anyway, there will be no commenting on the website so we can advertise it on scratch. Probably just muting for making tutorials, editing other people's tutorials, and banning from the website.

**Comment by markverb1** [comment]
oh yeah... The st. So ummmm,we __have__ to moderate tutorials before they are public so the st doesnt kill us

**Comment by TheColaber** [comment]
Nah, using the same moderation strategy that the st uses should be fine. We can just moderate when we can and check reported tutorials.
